package.json

```diff
@@ -11,7 +11,7 @@
         "build:docs": "lerna run --scope @fremtind/portal build --stream",
         "commit": "git-cz",
         "predeploy:docs": "yarn run build:docs",
-        "deploy:docs": "gh-pages -d portal/public",
+        "deploy:docs": "gh-pages --add --dist portal/public",
```

Comment:
Legger til `--add` for å unngå å slette previews ved deploy av main, om vi har flere parallelle. Ulempe er at gh-pages etter hvert vil kunne få en del _cruft_ i gamle filer.